src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
because i need a random value i replaced all the 
rand.gauss() functions with
arma::randu(1) 
but that doesnt works